QUARKUS-5614

```diff
@@ -0,0 +1,66 @@
+These tests will cover various scenarios of meta-annotations defined on both methods and classes.
+Furthermore, the tests will validate the combined usage of directly applied `@PermissionsAllowed` annotations and those defined through meta-annotations on a single method, ensuring they operate together.
+Additionally, the direct application of `@PermissionsAllowed` on methods and classes will be tested, as this is currently not covered by QE.
+
+
+## Automated test development
+
+### Definition of roles and permissions
+
+The tests will use 4 roles (for 4 users) and 4 defined permission.
+Each role will be assigned a unique set of permissions.
+These roles will be defined in property file using Elytron security.
+The need for 4 roles and 4 permission is to validate multiple combination and definition.
+
+### Test cases
+- Both definition on meta-annotation and direct use of `@PermissionsAllowed` will have same cases, which should have same results.
+- These cases are:
+  - Two annotation on one method where each have one permission
+  - Two annotation on one method where one have one permission and second have multiple permission
+  - Two annotation on one method where one have one permission and second have multiple permission and defined inclusive (user role must have all permission)
+  - One annotation with multiple permission
+  - One annotation with multiple permission and defined inclusive
+  - Permission inherted from definition in interface
+    - Two annotation on one method where one have one permission and second have multiple permission
+    - One annotation with multiple permission and defined inclusive
+  - Permission with definition of own permission class
+    - One annotation with multiple permission and custom permission class
+    - One annotation with multiple permission, custom permission class and defined inclusive
+  - Permission defined on class
+    - Method inhert defined permission
+    - Method inhert defined permission with defined inclusive
+    - Method override class permission by own definition
+- Testing direct use of `@PermissionsAllowed` and `@PermissionsAllowed` defined using meta-annotation together:
+  -  Two annotation on one method where:
+    - Dirrect usage have one permission and meta-annotation have multiple permission
+    - Dirrect usage have one permission and meta-annotation have multiple permission with defined inclusive
+    - Dirrect usage have multiple permissions and meta-annotation have one permission
+    - Dirrect usage have multiple permissions with defined inclusive and meta-annotation have one permission
+  - Permission inherted from definition in interface
+    - Dirrect usage have one permission and meta-annotation have multiple permission
+    - Dirrect usage have one permission and meta-annotation have multiple permission with defined inclusive
+
+
+### Impact on TS and resources
+The new module `security/permissions` will be added to QE TS.
```

Comment:
Sadly impl. little differs between REST stacks (Quarkus REST, RESTEasy, WS Next), I am perfectly fine if you are going to test this only with Quarkus REST, but you should specify what you are going to test here.

Reply:
Added it to the scope of testing part